src/containers/Notification/style.scss

```diff
@@ -0,0 +1,66 @@
+// stylesheet for notification
+.notification-wapper{
+    max-width: 593px;
+    margin:40px auto;
+    background-color: #fff;
+    box-shadow: 0px 0px 20px #00000005;
+    border-radius: 5px;
+    box-shadow: 0px 0px 99px #00000021;
+    border-radius: 5px;
+}
+
+.notification-area{
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 1px solid #f2f0f0;
+    margin: 0px 20px;
+    padding: 10px 0px;
+}
+
+.user-detail{
+    display: flex;
+    align-items: center;
+
+    img{
+        width: 47px;
+        height: 47px;
+        border-radius: 50%;
+        margin-right: 10px;
+    }
+
+    .user-icons{
+        width: 47px;
+        height: 47px;
+        border-radius: 50%;
+        margin-right: 10px;
+        background: #CDECF2 0% 0% no-repeat padding-box;
+        font: normal normal 600 14px/20px Open Sans;
+        letter-spacing: 0px;
+        color: #6B9CA5;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+
+    p{
+    font: normal normal normal 14px/20px Open Sans;
+    letter-spacing: 0px;
+    color: #504F4F;
+    width:85%;
+    }
+}
+.timer{
```

Comment:
Add space before open bracket
Add blank line on the above

Reply:
done